backend/person/models.py

```diff
@@ -150,3 +152,31 @@ class Meta:
 
     def __str__(self):
         return f"{self.agent} as {self.status_marker}"
+
+
+class Agent(Historical, AgentBase):
+    """
+    An aggregate model that represents an agent (person or group) in history. This model is based on one or multiple SourceDescriptions and other sources that are not part of this database.
+    """
+
+    pass
+
+
+class AgentDescription(SourceDescription, AgentBase):
```

Comment:
This is also applies to other models. In most cases, information that is represented in the `SourceDescription` does not need to be duplicated in the corresponding `Historical` model. It's more precise to display or query the agent through its descriptions.

I would say that an `Agent` may also include information of its own, but mainly things that are not really connected to sources, and more about representing the agent in the interface.

For example, the `Agent` Radegund could include a name, longer description, and an image that can be used in the interface. Then an `AgentDescription` of Radegund describes the language used in source texts to describe her gender, social status, names used in the text, etc.